puppetfactory/puppetfactory.gemspec

```diff
@@ -3,7 +3,7 @@ require 'date'
 
 Gem::Specification.new do |s|
   s.name              = "puppetfactory"
-  s.version           = '0.5.5'
+  s.version           = '0.5.6'
```

Comment:
Please don't mix version bumps with functionality commits. It makes it super super hard to back them out if we ever need to do so.